home/models.py

```diff
@@ -93,6 +93,9 @@ def get_context(self, request):
         # Serialize sites with the results of their latest scans for the teaser
         context['sites_json'] = json.dumps([site.to_dict() for site in sites])
 
+        regions = Region.objects.all()
```

Comment:
I'm wondering if we ought to be showing all regions by default? While I was playing around with this locally I got into a situation where I had added a region to some Sites but then removed it from them. I couldn't find a way to delete (or otherwise hide) a region directly and it was still on the home page even though clicking through led to an empty leaderboard. 

It's possible we won't get into that situation on the live site due to careful region management. What would you think, though, about having this query only fetch regions that have at least one site belonging to them?

Reply:
So the way it was previously implemented and the way I thought is if there is a region, irrespective of whether it has a Site under it, I am showing it. But can change to filter on the basis of whether the region at least has one site.  I think that will be good and make space for more regions if some region doesn't have Sites